Framework/DataHandling/test/LoadILLDiffractionTest.h

```diff
@@ -493,6 +494,65 @@ class LoadILLDiffractionTest : public CxxTest::TestSuite {
     checkTimeFormat(outputWS);
   }
 
+  void test_IN5_omega_scan() {
```

Comment:
Can you add tests also for the PANTHER and SHARP numors?

Reply:
Not a problem for PANTHER, but there was no SHARP scan mode data the last time I checked. Do you know whether it was used in the last cycle?

Reply:
You just added in the test data. I think it's 000104.

Reply:
Oh yes, absolutely! It's exactly that one.